l3backend/l3backend-pdf.dtx

```diff
@@ -571,38 +571,35 @@
 % \end{macro}
 % \end{macro}
 %
-% \begin{macro}{\@makecol@hook}
-%   Hooks to allow link breaking: something will be needed in format mode
-%   at some stage. At present this code is disabled as there is an open
-%   question about the name of the hook: to be resolved at the \LaTeXe{}
-%   end.
+% \begin{macro}{\@makecol}
+%   Install the necessary code to break links.
 %    \begin{macrocode}
-\use_none:n
+\bool_lazy_and:nnT
+  { \cs_if_exist_p:N \hook_gput_code:nnn }
+  { \cs_if_exist_p:N \@makecol }
   {
-    \cs_if_exist:NT \@makecol@hook
+    \hook_gput_code:nnn { cmd / @makecol / before }
+      { backend }
       {
-        \tl_put_right:Nn \@makecol@hook
+        \box_if_empty:NF \l_shipout_box
           {
```

Comment:
but I don't understand why \l_shipout_box should be non-empty at this point. I presume it is correct, but could there be perhaps a little explanation?

Reply:
This may need re-working: I suspect it's got to be hard-coded box 255 at present.